src/Features/Core/Portable/InlineHints/IInlineTypeHintsService.cs

```diff
@@ -17,6 +17,12 @@ namespace Microsoft.CodeAnalysis.InlineHints
     /// </summary>
     internal interface IInlineTypeHintsService : ILanguageService
     {
-        Task<ImmutableArray<InlineHint>> GetInlineHintsAsync(Document document, TextSpan textSpan, InlineTypeHintsOptions options, SymbolDescriptionOptions displayOptions, CancellationToken cancellationToken);
+        Task<ImmutableArray<InlineHint>> GetInlineHintsAsync(
+            Document document,
+            TextSpan textSpan,
+            InlineTypeHintsOptions options,
+            SymbolDescriptionOptions displayOptions,
+            bool displayAllOverride,
```

Comment:
@tmat i could also make this part of `InlineTypeHintsOptions`.  It just wouldn't be populated from any options-service, but would be provided directly by the feature. However, i wasn't certain if that was a pattern we wanted for our immutable-options/options-services in the future.  So, for now, i pulled out 'displayAllOverride' as a separate explicit option that is passed along.

If it is fine to move into InlineTypeHintsOptions , i'm happy to move into it.  BUt i wanted to check with you.